### PR TITLE
fix: add Harsh Singh

### PR DESCRIPTION
**Pull request by harshsinghatz**
Add harshsinghatz.json file with social information in public/data

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/889"></a>

### CODE REVIEW

**Review by krishguptadev — CHANGES_REQUESTED** [review]
You need to revert `package-lock.json`

File: public/data/harshsinghatz.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Harsh Singh",
+  "bio": "I am an Aspiring full stack developer. I am proficient with JavaScript and looking to do open source contributions frequently",
```
Comment (krishguptadev) [inline public/data/harshsinghatz.json:3]:
```suggestion
  "bio": "I am an Aspiring full stack developer. I am proficient with JavaScript and looking to do open source contributions frequently!",
```

Comment (syedsohan) [inline public/data/harshsinghatz.json:3]:
```suggestion
  "bio": "I am an aspiring full stack developer. I am proficient with JavaScript and looking forward to doing open source contributions frequently!",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Please revert the changes in the `package-lock.json` file. Let us know if you need any help! 😃

File: public/data/harshsinghatz.json (hunk 1 above)
Comment (krishguptadev) [reply public/data/harshsinghatz.json:3]: I am unsure why you marked that comment as resolved, it is a nice suggestion!

```suggestion
  "bio": "I am an aspiring full stack developer. I am proficient with JavaScript and looking to do open source contributions frequently!",
```

**Comment by harshsinghatz** [comment]
I have done the changes in the public/data/harshsinghatz.json but am not sure how to revert the package-lock.json

**Comment by ssensalo** [comment]
You can copy the current `package-lock.json` from the main branch into your branch and replace then commit the changes

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Your profile looks good. Please remove the package-lock file from the commit then we can get this merged :)

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**